### PR TITLE
DOC-3516 / DOC-4238

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/ROOT/pages/compatibility-versions-features.adoc [hunk 1]
```diff
@@ -15,91 +15,130 @@ This combination is not recommended by our technical support organization.
...
+| *✔*
 |===
 
+Note the https://www.couchbase.com/support-policy[End of Life dates^] for Couchbase Server and SDK versions.
+See the notes there for Support details.
+
 == Couchbase Feature Availability Matrix
 
-.Couchbase Node.js SDK and Couchbase Server Feature Availability Matrix
+To take advantage of all features offered by Couchbase Server, you need to know what version of the Java SDK provides compatibility for the features you want to use.
```
Comment (brett19) [inline modules/ROOT/pages/compatibility-versions-features.adoc]: Java? :D

File: modules/ROOT/pages/compatibility-versions-features.adoc [hunk 2]
```diff
@@ -15,91 +15,130 @@ This combination is not recommended by our technical support organization.
...
+4+| All Versions
 
 | SSL Connectivity
-3+| Not Supported
-6+| Since Node.js SDK 2.0
+4+| All Versions
 
 | Bulk Operations
-9+| NOT SUPPORTED IN NODEJS, except for Bulk "Get" Since Node.js SDK 2.0
+4+| NOT SUPPORTED IN NODEJS, except for Bulk "Get" Since Node.js SDK 2.0
```
Comment (brett19) [inline modules/ROOT/pages/compatibility-versions-features.adoc]: Do we really want to use caps like this?

File: modules/ROOT/pages/compatibility-versions-features.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/ROOT/pages/compatibility-versions-features.adoc]: Oops :)

File: modules/ROOT/pages/compatibility-versions-features.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/ROOT/pages/compatibility-versions-features.adoc]: Indeed we don't :+1:

**Review by brett19 — APPROVED** [review]

**Pull request merged**